internal/install/install.go

```diff
@@ -89,6 +98,21 @@ func ServiceLogsDir() (string, error) {
 	return filepath.Join(configurationDir, serviceLogsDir), nil
 }
 
+// ServiceDeployerComposeFile function returns the path to the selected service deployer's Docker Compose definition.
+func ServiceDeployerComposeFile(deployerName string) (string, error) {
```

Comment:
Nit: I think you can just call this `TerraformDeployerComposeFile` and remove the `deployerName` argument. If/when we have more deployers that use a docker compose file for setup, we can generalize this then.

Reply:
Fixed